content/docs/atom-guide/dev-guide/shaders/azsl/unbounded-arrays.md

```diff
@@ -0,0 +1,306 @@
+---
+linktitle: Unbounded Arrays in AZSL
+title: AZSL, Binding Rules For Unbounded Arrays
+description: There are limitations into how many and what kind of Unbounded Arrays can be declared. 
+weight: 100
+---
+AZSL supports declaration of Unbounded Arrays inside `ShaderResourceGroup`s, but there are limitations into how many and what kind of Unbounded Arrays can be declared. The shader compiler, **AZSLc**, makes sure those rules are not violated.  
+  
+Here is a summary of the rules and limitations per platform (DX12, Vulkan and Metal). '--use-spaces' and '--unique-idx' are command line arguments for **AZSLc** that are used for each platform.  
+| --use-spaces   | --unique-idx | platform | Results |
+|----------------|--------------|----------|---------|
+| OFF | OFF |                   | Only the last SRG can contain one last unbounded array for each:<br> t[], u[], b[], s[]|
+| OFF | ON  |                   | Only the last SRG can contain one, and only one, last unbounded array across all resource types:<br>Only one of t[], u[], b[], s[]|
+| ON  | OFF | DX12              | Each SRG can contain one last unbounded array for each: <br> t[], u[], b[], s[]|
+| ON  | ON  | Vulkan<br> Metal  | Each SRG can contain one, and only one, last unbounded array across all resource types: <br> Only one of t[], u[], b[], s[]|
+  
+## Things to know before going over some examples:
```

Comment:
```suggestion
## Things to know

To understand the examples on this page, review the following information:
```
* Shorten the title header.
* Move your explanation of this section into the first sentence.

Reply:
Fixed.